fastlane/Fastfile

```diff
@@ -138,9 +138,11 @@ platform :ios do
   desc 'Creates a new release branch from the current develop'
   lane :complete_code_freeze do |options|
     ios_completecodefreeze_prechecks(options)
-    ios_localize_project
-    version = ios_get_app_version
-    trigger_beta_build(branch_to_build: "release/#{version}")
+    generate_strings_file_for_glotpress
+
+    UI.confirm('Ready to push changes to remote and trigger the beta build?') unless ENV['RELEASE_TOOLKIT_SKIP_PUSH_CONFIRM']
+    push_to_git_remote  
```

Comment:
As discussed in Slack, I've now move the responsibility for doing the `git push` to the Fastfile, with an optional confirmation, as opposed to making the action do the push behind our back.

We initially mentioned creating a dedicated action to do the commit and push stuff, but after discovering that those actions existed built-in in `fastlane`, I decided to go with the built-in ones like here and [below](https://github.com/wordpress-mobile/WordPress-iOS/pull/17519/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R432).